### PR TITLE
New Port: Jump n Bump

### PR DESCRIPTION
**Pull request by monkeyx-net**
# New Port for Jump n Bump
## Game Information
- **Title**: Jump n Bump
- **URL**:  https://github.com/fabiangreffrath/jumpnbump

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x ] AmberELEC
- [x ] ArkOS
- [ x] JELOS

### Resolution Tests
Test all major resolutions:
- [ x] 480x320
- [x ] 640x480
- [x ] 720x720 (RGB30)
- [x ] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by Cebion — APPROVED** [review]
:+1:

File: ports/jumpnbump/README.md
Comment (tabreturn) [inline ports/jumpnbump/README.md:1]: L1 and L2 appear mapped to keys but aren't listed in the Controls here

**Review by tabreturn — CHANGES_REQUESTED** [review]

**Review by kloptops — APPROVED** [review]
👍

**Comment by Cebion** [comment]
F10 toggles fullscreen
ESC ends the game
F12 closes the game and quits

Has no relevance.

**Pull request merged**